src/C/gdal.jl

```diff
@@ -766,118 +806,99 @@ end
 
 Set single metadata item.
 """
-function GDALSetMetadataItem(arg1::GDALMajorObjectH, arg2, arg3, arg4)
+function GDALSetMetadataItem(arg1, arg2, arg3, arg4)
     ccall((:GDALSetMetadataItem, libgdal), CPLErr, (GDALMajorObjectH, Cstring, Cstring, Cstring), arg1, arg2, arg3, arg4)
 end
 
-
 """
     GDALGetDescription(GDALMajorObjectH hObject) -> const char *
 
 Fetch object description.
 """
-function GDALGetDescription(arg1::GDALMajorObjectH)
+function GDALGetDescription(arg1)
     ccall((:GDALGetDescription, libgdal), Cstring, (GDALMajorObjectH,), arg1)
 end
 
-
 """
     GDALSetDescription(GDALMajorObjectH hObject,
                        const char * pszNewDesc) -> void
 
 Set object description.
 """
-function GDALSetDescription(arg1::GDALMajorObjectH, arg2)
+function GDALSetDescription(arg1, arg2)
     ccall((:GDALSetDescription, libgdal), Cvoid, (GDALMajorObjectH, Cstring), arg1, arg2)
 end
 
-
 """
     GDALGetDatasetDriver(GDALDatasetH hDataset) -> GDALDriverH
 
 Fetch the driver to which this dataset relates.
 """
-function GDALGetDatasetDriver(arg1::GDALDatasetH)
+function GDALGetDatasetDriver(arg1)
     ccall((:GDALGetDatasetDriver, libgdal), GDALDriverH, (GDALDatasetH,), arg1)
 end
 
-
 """
     GDALGetFileList(GDALDatasetH hDS) -> char **
 
 Fetch files forming dataset.
 """
-function GDALGetFileList(arg1::GDALDatasetH)
+function GDALGetFileList(arg1)
     ccall((:GDALGetFileList, libgdal), Ptr{Cstring}, (GDALDatasetH,), arg1)
 end
 
-
-"""
-    GDALClose(GDALDatasetH hDS) -> friend void
-
-Close GDAL dataset.
-
-### Parameters
-* **hDS**: The dataset to close. May be cast from a "GDALDataset *".
-"""
-function GDALClose(arg1::GDALDatasetH)
+function GDALClose(arg1)
```

Comment:
did we lose the docstring here?

Reply:
Good catch, fixed in 5e8633e7a4ab6d834bb4c8cd356427a6f54c2113, I picked the wrong XML node with name GDALClose here.